libraries/AC_AutoTune/AC_AutoTune.h

```diff
@@ -67,44 +92,121 @@ class AC_AutoTune {
 
     // internal init function, should be called from init()
     bool init_internals(bool use_poshold,
-                        AC_AttitudeControl_Multi *attitude_control,
+                        AC_AttitudeControl *attitude_control,
                         AC_PosControl *pos_control,
                         AP_AHRS_View *ahrs_view,
                         AP_InertialNav *inertial_nav);
 
     // initialise position controller
     bool init_position_controller();
 
-private:
+    // main state machine to level vehicle, perform a test and update gains
```

Comment:
I think that "control_attitude()" can remain "private:" (as it was before) because it is only called from within AC_AutoTune.cpp.

Reply:
made all methods only used in autotune.cpp private